scylla/src/transport/session.rs

```diff
@@ -1593,7 +1593,7 @@ impl Session {
                             &execution_profile,
                             ExecuteQueryContext {
                                 is_idempotent: statement_config.is_idempotent,
-                                consistency: statement_config.consistency,
+                                consistency_set_on_statement: statement_config.consistency,
```

Comment:
> As there happens a lot with consistencies in session's methods that
> execute requests, proper naming is crucial. Therefore,
> ExecuteQueryContext's field is renamed from the vague `consistency` to
> `consistency_set_on_statement` (the one directly set), as it is exactly
> its semantics.

I don't understand the first sentence and the implication that follows it.

Reply:
Let me rephrase it this way:

> There are three levels of consistency that are taken into account by the `Session`: default exec profile's, per-statement exec profile's and per-statement one. To avoid confusion, we should use naming that clearly distinguishes them. Therefore, ...

Is it clear now?

Reply:
Yes, sounds better.